src/client/src/components/Login/Login.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './Login.css';
+
+
+function checkLoginResponse(response) {
+    let gotError = !response.ok;
+    if (gotError) {
+        throw new Error("Unable to log in - check username and password");
+    }
+    return response
+}
+
+
+async function loginUser(credentials) {
```

Comment:
we are working with async await as a convension. can change that, or leave a todo to update it later. more readable and easier when it's standardized through out the app